### PR TITLE
Grammatically updated the CONTRIBUTING file

### PR DESCRIPTION
**Pull request by Coolsum4**
## Describe your changes
Small grammatical update to CONTRIBUTING file

## Checklist before requesting a review
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```
Comment (vladklokun) [inline CODE_OF_CONDUCT.md:18]: issue: adding a trailing _-s_ in “includes” is a mistake

In Present Simple, plural nouns (“examples”) drop the trailing _-s_ in the associated verb (“include”). The correct form is “Examples [...] include”, as it currently is.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ community include:
 * Focusing on what is best not just for us as individuals, but for the
   overall community
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behavior includes:
```
Comment (vladklokun) [inline CODE_OF_CONDUCT.md:28]: issue: same as above

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ decisions when appropriate.
 
 ## Scope
 
-This Code of Conduct applies within all community spaces, and also applies when
+This Code of Conduct applies within all community spaces and also applies when
```
Comment (vladklokun) [inline CODE_OF_CONDUCT.md:53]: issue: placing a comma between two independent clauses is okay, no change needed

**Comment by vladklokun** [comment]
Thanks for taking an interest in contributing to Kubescape!

There are no changes needed to be made to the files in the PR, so I will be closing this Pull Request. Have a nice day :)

**Comment by Coolsum4** [comment]
ok thank you for replying

**Pull request closed without merging**